apps/spruce/src/analytics/addPageAction.ts

```diff
@@ -34,8 +34,22 @@ interface RequiredProperties {
   object: AnalyticsObject;
 }
 
+type ActionTypePrefixes =
+  | "Changed"
+  | "Clicked"
+  | "Created"
+  | "Deleted"
+  | "Redirected"
+  | "Filtered"
+  | "Saved"
+  | "Sorted"
+  | "Toggled"
+  | "Viewed"
+  | "Used"
+  | "System Event";
```

Comment:
This doesn't seem to be used anywhere, should it still be included?

Reply:
It's used in Parsley so I just reused them. Later on in this project I plan on combining the analytics utils and moving them into the lib folder.